frontend/src/components/dashboard/SelectFilter.vue

```diff
@@ -24,14 +24,19 @@
         dense
         :input-value="item.selected"
         color="primary"
-        @click.prevent="toggle(item.value)"
+        @click.prevent="toggle(item.value, item.exclusiveNone)"
       >
         <v-list-item-title>
-          <slot name="item" v-bind="{ item, self }">{{ item.text }} </slot>
+          <slot name="item" v-bind="{ item, self }">{{ item.text }}</slot>
         </v-list-item-title>
-        <v-list-item-action v-if="multiple">
+        <v-list-item-action v-if="multiple && !item.exclusiveNone">
           <v-checkbox v-model="item.selected" dense />
         </v-list-item-action>
+        <v-list-item-action v-if="item.exclusiveNone">
+          <v-radio-group v-model="item.selected">
+            <v-radio dense :value="true" />
```

Comment:
At first glance, I was confused why this was a radiobox for responsibles, but a checkbox for status. But then I recalled the discussion we had with OR and AND selection 🙄 

Let's see how the users cope with it and whether we receive any feedback on it.